tests/test_timer.cpp

```diff
@@ -37,116 +37,112 @@ struct Timer : public testing::Test
     std::atomic_bool timedout = false;
     std::condition_variable cv;
     std::mutex cv_m;
+    const std::chrono::milliseconds default_timeout{10};
+    const std::chrono::milliseconds test_sleep{20};
+    const std::chrono::seconds default_wait{1};
 };
 } // namespace
 
 TEST_F(Timer, times_out)
 {
-    mpu::Timer t{duration_cast<seconds>(1ms), [this]() {
-                     {
-                         std::lock_guard<std::mutex> lk{cv_m};
-                         timedout.store(true);
-                     }
+    mpu::Timer t{default_timeout, [this]()
+                 {
+                     timedout.store(true);
                      cv.notify_all();
                  }};
-    std::unique_lock<std::mutex> lk(cv_m);
 
     t.start();
     ASSERT_FALSE(timedout.load()) << "Should not have timed out yet";
 
-    cv.wait_for(lk, 10ms); // Windows CI needs longer...
+    std::unique_lock<std::mutex> lk(cv_m);
```

Comment:
This is not doing anything now, right?

Reply:
The only thing it does is allow the main thread to wait on the condition variable and get notified when the callback completes so as not to make the test run any longer than necessary.